setup.py

```diff
@@ -31,30 +35,8 @@
     keywords = "Earley LALR parser parsing ast",
     url = "https://github.com/erezsh/lark",
     download_url = "https://github.com/erezsh/lark/tarball/master",
-    long_description='''
```

Comment:
That's a nice idea, but I think lark's readme is too long for this field, and might not display correctly in non-web situations.

But I'm open to be persuaded.